### ISSUE TITLE
Link hover status bar covers input box

### ISSUE THREAD

**Issue by lcarva** [thelounge/thelounge#2239]
When hovering over links, the url is displayed in the status bar at the bottom left. This is expected. However, I often find myself in a situation where my mouse cursor is somewhere in the middle of the screen while I'm typing a message, and as messages in channel scroll by, eventually a link ends up under my mouse cursor. At this point the input box is mostly covered by browser's status bar requiring the mouse cursor to be moved out of the way.

I don't think the status bar behavior can, nor should, be changed. Maybe we can re-design some of it so the message input field is moved a few pixels up?

In the mean time there's probably some custom CSS I can apply to just move it up, but this will likely look odd without a proper design.

**Comment by lcarva** [comment]
Looks ugly, but does the tricky:
```css
#chat-container #form { padding-bottom: 25px}
```

**Comment by McInkay** [comment]
So, I'm not sure we want to do something in core to deal with this.

Custom css or themes are fine to deal with it, but realistically, anything under that point will be covered by the status bar and I definitely don't want it just being blank.

**Comment by astorije** [comment]
Please don't hit me.

This is purely for archive sake and not at all trying to compare or justify anything, but I have noticed Slack avoids that issue with crazy-ass margins:

<img width="501" alt="screen shot 2018-03-30 at 01 13 52" src="https://user-images.githubusercontent.com/113730/38126234-3845b402-33bd-11e8-9a40-e7f6c33b6f55.png">

I don't like how it specifically looks for them btw, but I thought it was worth mentioning.

**Comment by dgw** [comment]
I'm with @McInkay, in that I believe it shouldn't be The Lounge's responsibility to handle browser-specific GUI issues like this. If Chrome (for example) had a hypothetical pseudo-element that let one move/restyle its status bar, that would be one thing, but The Lounge's GUI shouldn't have to change just to accommodate this when keeping the mouse cursor out of the chat unless you're clicking/scrolling will do.

And honestly, I don't like that Slack example either. It's kind of ugly (but I won't hit you, @astorije :grin:).

**Comment by astorije** [comment]
I totally agree, @dgw. I guess my mindset on this is that if it takes an extra 2px to make this look good, then it's an acceptable cost. Whether we like it or not, we have to be pleasant to use on the main browsers if we can, which is why we often make compromises or bug fixes to comply with Edge, Safari, etc.

That said, I think we're all in agreement here, and it actually looks decent on default theme (see below). As long as the default theme work well with these kind of things, I'm happy (same argument as why default theme has nick editor, channel close button, join channel form, and other things advanced users prefer to hide).

<img width="492" alt="screen shot 2018-03-30 at 02 35 19" src="https://user-images.githubusercontent.com/113730/38127252-542d3b08-33c3-11e8-984f-e1f863140583.png">

And yeah, it does look ugly, doesn't it. Slack made it usable, but they forgot to make it look good :D This is where we steal market shares 😂
